validator/main.py

```diff
@@ -332,6 +335,14 @@ def validate(
         else:
             raise ValueError("Either classifier or llm must be enabled.")
 
+        model_input = {
+            "text": value,
+            "valid_topics": self._valid_topics,
+            "invalid_topics": self._invalid_topics
+        }
+
+        found_topics = self._inference(model_input)
```

Comment:
On second glance, this line doesn't look right.  This should either only be called when utilizing the zero shot model, or we need to override `Validator._inference` to use the logic above that distinguishes between the three options.

Reply:
The previous if checks need to come back.  Otherwise using only an llm isn't enabled